src/test/java/org/apache/mesos/offer/MesosResourcePoolTest.java

```diff
@@ -1,41 +1,28 @@
 package org.apache.mesos.offer;
 
-import java.util.*;
-
 import org.apache.mesos.Protos;
 import org.apache.mesos.Protos.Offer;
 import org.apache.mesos.Protos.Resource;
-
-import org.apache.mesos.protobuf.OfferBuilder;
-import org.apache.mesos.protobuf.ResourceBuilder;
-
-import org.apache.mesos.protobuf.ValueUtils;
+import org.apache.mesos.testutils.OfferTestUtils;
+import org.apache.mesos.testutils.ResourceTestUtils;
 import org.junit.Assert;
 import org.junit.Test;
 
-public class MesosResourcePoolTest {
+import java.util.Arrays;
 
-  private static final String testOfferId = "test-offer-id";
-  private static final String testFrameworkId = "test-framework-id";
-  private static final String testSlaveId = "test-slave-id";
-  private static final String testHostname = "test-hostname";
-  private static final String testMountRoot = "test-mount-root";
-  private static final String testRole = "test-role";
-  private static final String testPrincipal = "test-principal";
-  private static final String testContainerPath = "test-containter-path";
-  private static final String testResourceId = "test-resource-id";
+public class MesosResourcePoolTest {
 
   @Test
   public void testEmptyUnreservedAtomicPool() {
-    Offer offer = getOffer(ResourceBuilder.cpus(1.0));
+    Offer offer = OfferTestUtils.getOffer(ResourceTestUtils.getUnreservedCpu(1.0));
```

Comment:
This library is constantly attempting to reserver resources.  It's hard to say what's the default for us.  I like being explicit that something isn't reserved yet, with the implication that it's about to be probably.